### PR TITLE
Added tlg0093.tlg004.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
De lapidibus

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |       +1 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +4295 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg004/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg004/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3959)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |       +1 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +4295 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg004/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg004/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3965)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |       +1 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +4295 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg004/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg004/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3972)*

File: data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,664 @@
...
+      <ref target="http://gallica.bnf.fr/ark:/12148/bpt6k28209z">Gallica</ref>
+     </biblStruct>
+    </listBibl>
+   </sourceDesc>
+  </fileDesc>
+  <encodingDesc>
+   <p>Text encoded in accordance with the latest EpiDoc standards</p>
+   <refsDecl n="CTS">
+    <cRefPattern matchPattern="(.+)" n="chapter"
+     replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
```
Comment (planatheisa) [inline data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml]: Wouldn't this get the fragment-div instead of the chapter-divs?

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +6 |
| `metadata_total`   |       +6 |
| `nodes_count`      |     +134 |
| `texts_passing`    |       +6 |
| `texts_total`      |       +6 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +15509 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg004/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | New      |
| `./data/tlg2042/tlg069/__cts__.xml`                   | New      |
| `./data/tlg2042/tlg069/tlg2042.tlg069.1st1K-grc1.xml` | New      |
| `./data/tlg2042/tlg070/__cts__.xml`                   | New      |
| `./data/tlg2042/tlg070/tlg2042.tlg070.1st1K-grc1.xml` | New      |
| `./data/tlg2042/tlg071/__cts__.xml`                   | New      |
| `./data/tlg2042/tlg071/tlg2042.tlg071.1st1K-grc1.xml` | New      |
| `./data/tlg2042/tlg072/__cts__.xml`                   | New      |
| `./data/tlg2042/tlg072/tlg2042.tlg072.1st1K-grc1.xml` | New      |
| `./data/tlg2042/tlg074/__cts__.xml`                   | New      |
| `./data/tlg2042/tlg074/tlg2042.tlg074.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg004/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg004/tlg0093.tlg004.1st1K-grc1.xml` | Passing  |
| `./data/tlg2042/tlg069/__cts__.xml`                   | Passing  |
| `./data/tlg2042/tlg069/tlg2042.tlg069.1st1K-grc1.xml` | Passing  |
| `./data/tlg2042/tlg070/__cts__.xml`                   | Passing  |
| `./data/tlg2042/tlg070/tlg2042.tlg070.1st1K-grc1.xml` | Passing  |
| `./data/tlg2042/tlg071/__cts__.xml`                   | Passing  |
| `./data/tlg2042/tlg071/tlg2042.tlg071.1st1K-grc1.xml` | Passing  |
| `./data/tlg2042/tlg072/__cts__.xml`                   | Passing  |
| `./data/tlg2042/tlg072/tlg2042.tlg072.1st1K-grc1.xml` | Passing  |
| `./data/tlg2042/tlg074/__cts__.xml`                   | Passing  |
| `./data/tlg2042/tlg074/tlg2042.tlg074.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/4012)*

**Pull request merged**